InfiniteScroll/InfiniteScroll/StoryViewController.swift

```diff
@@ -97,17 +100,33 @@ class StoryViewController: UIViewController {
 
 extension StoryViewController: UITableViewDelegate, UITableViewDataSource {
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return actualList.count
+        return actualList.count - 1
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         guard let cell = storyTableView.dequeueReusableCell(withIdentifier: "StoryTableViewCell", for: indexPath) as? StoryTableViewCell else { return UITableViewCell() }
         cell.initialSetup()
         cell.makeUI()
         cell.myLabel.text = "\(storyList[indexPath.row])"
+        let url = self.kakaoData[indexPath.row].thumbnail
+        let thumbnailURL = URL(string: url)
+        
+        DispatchQueue.global().async {
+            if let thumbnailURL = thumbnailURL {
+                let data = try? Data(contentsOf: thumbnailURL)
+                DispatchQueue.main.async {
+                    let thumbnail = UIImage(data: data!)
+                    cell.thumbnailImage.image = thumbnail
+                }
+            }
+        }
         
         return cell
     }
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
```

Comment:
해당 함수는 다음 작업에 사용할 예정이어서 선언해두신 듯 보입니다!
그럼에도 눈에 띈다면 가능한 컨벤션을 지켜주세요~!

```suggestion
    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) { }
```